### PR TITLE
chore(deps): Upgrade `geth` to `v1.13.1` 

### PR DESCRIPTION
**Pull request by itsdevbear**
TODO: `EIP-6780`

### CODE REVIEW

File: eth/core/state/journal/selfdestructs.go [hunk 1]
```diff
@@ -45,71 +45,76 @@ type suicideStatePlugin interface {
...
 	s.ssp.SubBalance(addr, s.ssp.GetBalance(addr))
 
 	// add to journal.
 	s.journal.Push(&addr)
-	return true
 }
 
-// HasSuicided implements the PolarisStateDB interface by returning if the contract was suicided
+func (s *selfDestructs) Selfdestruct6780(_ common.Address) {
+	// TODO: IMPLEMENT EIP-6780
```
Comment (ocnc) [inline eth/core/state/journal/selfdestructs.go:112]: wip?

Comment (ocnc) [reply eth/core/state/journal/selfdestructs.go:112]: nvm, just saw the desciption

File: e2e/hive/simulators/rpc-compat/tests/eth_estimateGas/estimate-simple-transfer.io [hunk 2]
```diff
@@ -1,2 +1,2 @@
 >> {"jsonrpc":"2.0","id":1,"method":"eth_estimateGas","params":[{"from":"0xaa00000000000000000000000000000000000000","to":"0x0100000000000000000000000000000000000000"}]}
-<< {"jsonrpc":"2.0","id":1,"result":"0x6270"}
+<< {"jsonrpc":"2.0","id":1,"result":"0x5a3c"}
```
Comment (itsdevbear) [inline e2e/hive/simulators/rpc-compat/tests/eth_estimateGas/estimate-simple-transfer.io:2]: reduce estimateGas overshoot from 20 to 10% thats the cause for this change.

Comment (itsdevbear) [reply e2e/hive/simulators/rpc-compat/tests/eth_estimateGas/estimate-simple-transfer.io]: nvm went back to 20%

File: eth/core/chain_writer.go [hunk 3]
```diff
@@ -77,15 +80,55 @@ func (bc *blockchain) Prepare(ctx context.Context, number uint64) {
 		Number:     new(big.Int).SetUint64(number),
 		GasLimit:   bc.gp.BlockGasLimit(),
 		Time:       timestamp,
-		BaseFee:    misc.CalcBaseFee(bc.Config(), parent),
 	}
 
+	// TODO: Settable in PrepareProposal.
```
Comment (itsdevbear) [inline eth/core/chain_writer.go:85]: @calbera @ocnc please review this section especially.

Comment (itsdevbear) [reply eth/core/chain_writer.go:85]: other than this change, PR is fairly unremarkable.

File: eth/polar/node.go [hunk 4]
```diff
@@ -37,6 +37,9 @@ func NewGethNetworkingStack(config *node.Config) (NetworkingStack, error) {
 		return nil, err
 	}
 
+	// In Polaris we don't use P2P at the geth level.
+	node.SetP2PDisabled(true)
```
Comment (itsdevbear) [inline eth/polar/node.go:41]: @calbera @ocnc disabling geth p2p as well.

**Review by ocnc — APPROVED** [review]
lgtm, make a ticket in issues for 6780 so we don't forget or for anyone who might want to take a stab at it

**Review by calbera — APPROVED** [review]

**Comment by itsdevbear** [comment]
Say my name, say my name
If you love me, let me hear you

**Pull request merged**